x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_issues_step/fix_issues_step.tsx

```diff
@@ -105,7 +105,7 @@ export const getFixIssuesStep = ({
           <p>
             <FormattedMessage
               id="xpack.upgradeAssistant.overview.fixIssuesStepDescription"
-              defaultMessage="You must resolve any critical Elasticsearch and Kibana configuration issues before upgrading to Elastic 8.x. Ignoring warnings might result in differences in behavior after you upgrade. {accessDeprecationLogsMessage}"
+              defaultMessage="You can start resolving any critical Elasticsearch and Kibana configuration issues before upgrading to the next major version of Elastic, ignoring warnings might result in differences in behavior. {accessDeprecationLogsMessage}"
```

Comment:
"...configuration issues to prepare the upgrade to ..."

Reply:
There are some grammatical issues with the current copy:

```suggestion
              defaultMessage="You can start resolving any critical Elasticsearch and Kibana configuration issues before upgrading to the next major version of Elastic. Ignoring warnings might result in differences in behavior. {accessDeprecationLogsMessage}"
```

I'd make the copy a bit more imperative:

```suggestion
              defaultMessage="Resolve any critical Elasticsearch and Kibana configuration issues before upgrading to the next major version. Ignoring warnings might result in unexpected behavior. {accessDeprecationLogsMessage}"
```